tests/console/validate_lvm_raid1.pm

```diff
@@ -36,8 +36,12 @@ sub run {
     _check_raid1_partitioning($config, $actual_num_devs);
     _remove_raid_disk($config, $actual_num_devs);
     _reboot();
-    $self->wait_boot;
-
+    if (is_opensuse) {
+        select_console('root-console');
```

Comment:
you will be selecting `root-console` twice, here and in `_check_raid_disks_after_reboot`.